chatmaild/src/chatmaild/metadata.py

```diff
@@ -117,7 +118,25 @@ def handle_dovecot_request(msg, transactions, notifier, metadata, iroh_relay=Non
         if keyname[0] == "priv" and keyname[2] == metadata.DEVICETOKEN_KEY:
             metadata.add_token_to_addr(addr, value)
         elif keyname[0] == "priv" and keyname[2] == "messagenew":
-            notifier.new_message_for_addr(addr, metadata)
+            proc = subprocess.run(
+                [
+                    "doveadm",
+                    "fetch",
+                    "-u",
+                    addr,
+                    "--",
+                    "hdr.auto-submitted",
+                    "uid",
+                    value,
+                ],
+                capture_output=True,
+                check=True,
+            )
```

Comment:
what happens if  "doveadm fetch" fails? Will this this not error out completely because `check==True` ?